### ISSUE TITLE
DOCS:  inconsistent reference to admin route

### ISSUE THREAD

**Issue by pgnd** [sonata-project/SonataAdminBundle#4890]
noting at AdminBundle install docs,

[**_Admin Bundle:Installation_**](https://sonata-project.org/bundles/admin/3-x/doc/getting_started/installation.html)

_1.5. Step 4: Import Routing Configuration_
```
routing.yml
	admin_area:
	    resource: "@SonataAdminBundle/Resources/config/routing/sonata_admin.xml"
	    prefix: /admin
```

On same page, 2nd section just below,

_1.7. Step 6: Define routes_
```
routing.yml
	admin:
	    resource: '@SonataAdminBundle/Resources/config/routing/sonata_admin.xml'
	    prefix: /admin

	_sonata_admin:
	    resource: .
	    type: sonata_admin
	    prefix: /admin
```

"**admin_area:**" vs "**admin:**" is unclear -- inconsistency or redundancy?

Should be made consistent.

**Comment by greg0ire** [comment]
I think it's inconsistency. Can you please make a PR for this?

**Comment by pgnd** [comment]
PR -> 
[fix doc routing section inconsistency #4891](https://github.com/sonata-project/SonataAdminBundle/pull/4891)
